app/ui/browser-modern/sagas/page-sagas.js

```diff
@@ -77,6 +81,32 @@ export function* destroyPageSession({ id, withUI }) {
   }
 }
 
+function* forkPage(pageId, historyIndex) {
+  const currentPageIndex = yield select(PageSelectors.getPageIndexById, pageId);
+  const historyURLs = yield select(PageSelectors.getPageHistoryURLs, pageId);
```

Comment:
This should throw an error and cause the state to not change -- should have tests for what happens in failing sagas with composed actions